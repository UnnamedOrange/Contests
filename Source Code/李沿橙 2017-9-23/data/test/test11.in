5
4
900 1000 5 900 
10
1 2 3 4 5 6 7 8 9 10 
5
1 3 5 4 3 
10
7 43 3 6 32 2 42 23 1 30 
10
136 369 386 461 524 582 730 817 877 884 
