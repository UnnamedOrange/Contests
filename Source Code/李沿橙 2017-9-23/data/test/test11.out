5 900 1000 900 
1 2 4 6 8 10 9 7 5 3 
1 3 4 5 3 
1 2 3 6 7 30 32 42 43 23 
136 369 461 524 730 877 884 817 582 386 
